core/scripts/vrfv2plus/testnet/util.go

```diff
@@ -0,0 +1,305 @@
+		e.Owner,
+		e.Ec,
+		common.HexToAddress(coordinatorAddress),
+		common.HexToAddress(linkAddress),
+		uint32(1000000), // gas callback
+		uint16(5),       // confs
+		uint32(1),       // words
+		keyHash,
+		nativePayment)
+	helpers.PanicErr(err)
+	return helpers.ConfirmContractDeployed(context.Background(), e.Ec, tx, e.ChainID), consumer
+}
+
+func eoaFundSubscription(e helpers.Environment,
+	coordinator vrf_coordinator_v2plus.VRFCoordinatorV2Plus, linkAddress string, amount *big.Int, subID uint64) {
+	linkToken, err := link_token_interface.NewLinkToken(common.HexToAddress(linkAddress), e.Ec)
+	helpers.PanicErr(err)
+	bal, err := linkToken.BalanceOf(nil, e.Owner.From)
+	helpers.PanicErr(err)
+	fmt.Println("Initial account balance:", bal, e.Owner.From.String(), "Funding amount:", amount.String())
+	b, err := utils.ABIEncode(`[{"type":"uint64"}]`, subID)
+	helpers.PanicErr(err)
+	e.Owner.GasLimit = 500000
+	tx, err := linkToken.TransferAndCall(e.Owner, coordinator.Address(), amount, b)
+	helpers.PanicErr(err)
+	helpers.ConfirmTXMined(context.Background(), e.Ec, tx, e.ChainID, fmt.Sprintf("sub ID: %d", subID))
+}
+
+func printCoordinatorConfig(coordinator *vrf_coordinator_v2plus.VRFCoordinatorV2Plus) {
+	cfg, err := coordinator.SConfig(nil)
+	helpers.PanicErr(err)
+
+	feeConfig, err := coordinator.SFeeConfig(nil)
+	helpers.PanicErr(err)
+
+	fmt.Printf("Coordinator config: %+v\n", cfg)
+	fmt.Printf("Coordinator fee config: %+v\n", feeConfig)
+}
+
+func setCoordinatorConfig(
+	e helpers.Environment,
+	coordinator vrf_coordinator_v2plus.VRFCoordinatorV2Plus,
+	minConfs uint16,
+	maxGasLimit uint32,
+	stalenessSeconds uint32,
+	gasAfterPayment uint32,
+	fallbackWeiPerUnitLink *big.Int,
+	feeConfig vrf_coordinator_v2plus.VRFCoordinatorV2PlusFeeConfig,
+) {
+	tx, err := coordinator.SetConfig(
+		e.Owner,
+		minConfs,               // minRequestConfirmations
+		maxGasLimit,            // max gas limit
+		stalenessSeconds,       // stalenessSeconds
+		gasAfterPayment,        // gasAfterPaymentCalculation
+		fallbackWeiPerUnitLink, // 0.01 eth per link fallbackLinkPrice
+		feeConfig,
+	)
+	helpers.PanicErr(err)
+	helpers.ConfirmTXMined(context.Background(), e.Ec, tx, e.ChainID)
+}
+
+func registerCoordinatorProvingKey(e helpers.Environment,
+	coordinator vrf_coordinator_v2plus.VRFCoordinatorV2Plus, uncompressed string, oracleAddress string) {
+	pubBytes, err := hex.DecodeString(uncompressed)
+	helpers.PanicErr(err)
+	pk, err := crypto.UnmarshalPubkey(pubBytes)
+	helpers.PanicErr(err)
+	tx, err := coordinator.RegisterProvingKey(e.Owner,
+		common.HexToAddress(oracleAddress),
+		[2]*big.Int{pk.X, pk.Y})
+	helpers.PanicErr(err)
+	helpers.ConfirmTXMined(
+		context.Background(),
+		e.Ec,
+		tx,
+		e.ChainID,
+		fmt.Sprintf("Uncompressed public key: %s,", uncompressed),
+		fmt.Sprintf("Oracle address: %s,", oracleAddress),
+	)
+}
+
+func getRlpHeaders(env helpers.Environment, blockNumbers []*big.Int) (headers [][]byte, err error) {
```

Comment:
Probably worth moving this to the helpers too, this will be identical b/w v2 and v2+.

Reply:
Sure, moved it